test/extensions/voting-rep.js

```diff
@@ -1405,11 +1406,15 @@ contract("Voting Reputation", (accounts) => {
       await voting.claimReward(motionId, 1, UINT256_MAX, USER0, YAY);
       await voting.claimReward(motionId, 1, UINT256_MAX, USER1, NAY);
 
+      let votingPayout = await voting.getVoterReward(motionId, new BN(user0Value.slice(2, 66), 16));
```

Comment:
~Mm, can we use `user0Influence` here? This slicing is peculiar, I don't like it.~

Just realized we don't introduce those variables until the token/hybrid PR. Can we just include the WAD amounts explicitly? User 0 is WAD, User 1 is WAD.muln(2).